docs/en/setup/configuration/profiling.md

```diff
@@ -85,8 +85,24 @@ Based on the above two data types, the following metrics are provided.
 |connect execute time|Histogram|nanosecond|The socket connect/accept with other server/client execute time histogram|
 |close execute time|Histogram|nanosecond|The socket close execute time histogram|
 
+##### HTTP/1.x Data
+
+| Name                        | Type      | Unit        | Description                                                               |
+|-----------------------------|-----------|-------------|---------------------------------------------------------------------------|
+| http1_request_cpm           | Counter   | count       | The HTTP request counter                                                  |
+| http1_response_status_cpm   | Counter   | count       | The count of per HTTP response code                                       |
+| http1_request_package_size  | Histogram | Byte size   | The request package size                                                  |
+| http1_response_package_size | Histogram | Byte size   | The response package size                                                 |
+| http1_client_duration       | Histogram | millisecond | The duration of single HTTP response on the client side                   |
+| http1_server_duration       | Histogram | millisecond | The duration of single HTTP response on the server side                   |
+
 ## Configuration
```

Comment:
I noticed on this page, there are two `Configuration`. Could you update to make them more clear?

Reply:
Updated, thanks.